.travis.yml

```diff
@@ -11,6 +11,12 @@ python:
     - 3.6
     - pypy
     - pypy3
+# Enable 3.7 without globally enabling sudo and dist: xenial for other build jobs
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Comment:
The usage of sudo is deprecated, cf
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Reply:
Removed!